src/Auxiliary/Aux_Check_Parameter.cpp

```
@@ -1151,6 +1157,13 @@ void Aux_Check_Parameter()
    if ( ELBDM_BASE_SPECTRAL && OPT__FIXUP_FLUX )
       Aux_Message( stderr, "WARNING : OPT__FIXUP_FLUX will not be applied to the base level when %s is on !!\n",
                    "ELBDM_BASE_SPECTRAL" );
+
+#  if ( WAVE_SCHEME == WAVE_GRAMFE )
+   if ( OPT__FIXUP_FLUX )
+      Aux_Error( stderr, "WARNING : OPT__FIXUP_FLUX will not be applied when %s is on !!\n",
+                   "WAVE_SCHEME == WAVE_GRAMFE" );
```

- [x] I prefer to raise an error instead of a warning here. If you agree, please also replace `stderr` with `ERROR_INFO` and remove `WARNING : `.
- [ ] I would suggest disabling `OPT__FIXUP_FLUX` by default for `WAVE_GRAMFE` in `Init_Load_Parameter()`.

---

@hyschive 
I did not raise an error at this point on purpose. The reason is that the hybrid scheme uses `OPT__FIXUP_FLUX` to ensure mass conservation on large scales even when used with `WAVE_GRAMFE`. If you prefer, we can throw an error in the regular mode and a warning for the hybrid scheme. What do you think?

---

@KunkelAlexander Agree